### DISCUSSION TITLE
How are closed source tools harmful for civil society? What about open source tools?

### DISCUSSION THREAD

**Discussion by lvalles10** [albertkun/21S-ASIAAM-191A#115] [Past Thinking Caps]
Closed source tools capitalize on data for corporate gain. In a capitalist society, this is seen as "business as usual," but the utilization of oftentimes private and intimate data to turn a profit is dangerous and can even be abusive in certain applications.  The commodification of data by closed source tools completely strips it of its potential to be empowering for local communities. In a society that now runs on data, closed source tools rob folks of their individual liberties and freedom when it comes to how their data is being used. In many ways, closed source tools are harmful for civil society in the way they remove agency from the typical user and their data.

Open source tools, though more empowering and revolutionary in our capitalist society, still carry a certain amount of potential danger. As mentioned in discussion this week, the utilization of open source tools by anyone who has access to a computer can open the door to both positive and negative intentions when it comes to how data is being used. Despite these dangers of intention and neutrality of data, however, it still remains that open source data allows for freedom and expression of personal choice when using data which for marginalized folks can be especially beneficial. Open source tools allow for a circumvention of the commodification of data.

**Comment by jsxu315** [comment]
I agree that closed-source tools leave an open door for corporations to exploit communities — Palantir's contracts with ICE and Facebook's data scandals over the past few years come to mind. I’ve seen some companies, like Google, do projects for social change, and while the projects themselves seem good, the impact they make seems like a small dent compared to the harm the company does in other endeavours. For example, Google's [AI for Social Good](https://ai.google/social-good/) flood forecasting and COVID-19 projects appear to do good work, but I wonder how much net benefit/harm the company really does for these communities, since the company’s business model generally commodifies personal data — maybe not these communities’ specifically, but on a broader scale.
